uint16 keyframe_id #the keyframe number
uint8 vin #vehicle identification number
uint8[] data #the scan context descriptor
float32[] pose #the pose with this keyframe
uint16 count #the number of points in the keyframe
